### PR TITLE
php 72 and 73 and composer install

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

**Comment by addshore** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/issues/2087

**Comment by addshore** [comment]
needs 3.3.1

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,17 @@
...
 sudo: false
 
-install: travis_retry composer require "wikibase/data-model=$DM" --prefer-source
+install: travis_retry composer install
 
 script: composer ci
 
 after_success:
-  - if [[ "`phpenv version-name`" != "7.1" ]]; then exit 0; fi
+  - if [[ "`phpenv version-name`" != "7.3" ]]; then exit 0; fi
```
Comment (Ladsgroup) [inline .travis.yml]: 7.2?

Comment (addshore) [reply .travis.yml]: Doesn't really matter as long as a single currently run version is used

Comment (JeroenDeDauw) [reply .travis.yml]: Indeed. Though higher is normally a little bit faster.

Comment (addshore) [reply .travis.yml]: Done here https://github.com/wmde/WikibaseDataModelSerialization/commit/2d1a4732a329d70fabbb09337e4d59924f6ef0bd and in all of our other libraries

**Pull request merged**